app/lib/websocket.js

```diff
@@ -0,0 +1,7 @@
+import websocket from "websocket-stream";
+
+export default function createWebsocket(opts) {
+  opts = opts || {};
+  const url = opts.url || "ws://10.0.2.2:3000";
```

Comment:
`url` seems like it should be mandatory, right? Maybe it should be an explicit arg and `throw` when it's not present?